GitUI/GitExtensionsDialog.Designer.cs

```diff
@@ -41,21 +41,23 @@ private void InitializeComponent()
             this.ControlsPanel.BackColor = System.Drawing.SystemColors.ControlLight;
             this.ControlsPanel.Dock = System.Windows.Forms.DockStyle.Bottom;
             this.ControlsPanel.FlowDirection = System.Windows.Forms.FlowDirection.RightToLeft;
-            this.ControlsPanel.Location = new System.Drawing.Point(0, 292);
-            this.ControlsPanel.MinimumSize = new System.Drawing.Size(0, 45);
+            this.ControlsPanel.Location = new System.Drawing.Point(0, 299);
+            this.ControlsPanel.Margin = new System.Windows.Forms.Padding(0);
+            this.ControlsPanel.MinimumSize = new System.Drawing.Size(0, 38);
             this.ControlsPanel.Name = "ControlsPanel";
-            this.ControlsPanel.Padding = new System.Windows.Forms.Padding(8);
-            this.ControlsPanel.Size = new System.Drawing.Size(553, 45);
+            this.ControlsPanel.Padding = new System.Windows.Forms.Padding(7);
```

Comment:
Why is padding set to 7, should it not be a multiple of 4?

Reply:
Not necessarily. There must be something else in play in the layout... I have no time or interest in debugging this right now.
This aligns the margins with the Task Dialog bottom panel.